background/providers/rss/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file Provides a class for RSS Feeds
+ * @TODO Finish me...
+ */
+
+'use strict';
```

Comment:
We should be able to drop `'use strict'` in all of our files as ES6 modules don't require strict mode and neither does the newer version of node.